<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weekly</class>
 <widget class="QWidget" name="weekly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weekly Quests</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Weekly Quest</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="Week">
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Reset_Week">
      <property name="text">
       <string>Reset Quests</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week1">
      <property name="text">
       <string>Clear 3 Chaos Beast</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week2">
      <property name="text">
       <string>Clear 6 Chaos Beast</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week3">
      <property name="text">
       <string>Perfrom 2 Ascension</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week4">
      <property name="text">
       <string>XP Memoria Card 10 times</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week5">
      <property name="text">
       <string>XP Symphogear Card 10 times</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week6">
      <property name="text">
       <string>Kill 300 Ennemies in Quest</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week7">
      <property name="text">
       <string>Clear 10 Multi Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week8">
      <property name="text">
       <string>Join 10 Multi Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week9">
      <property name="text">
       <string>Join 25 Multti Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week10">
      <property name="text">
       <string>Get 30 cards via Gacha</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Week11">
      <property name="text">
       <string>Clear 20 Arena Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Week2</sender>
   <signal>toggled(bool)</signal>
   <receiver>Week1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Week9</sender>
   <signal>toggled(bool)</signal>
   <receiver>Week8</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
